docs/src/main/paradox/discovery/kubernetes.md

```diff
@@ -19,25 +19,43 @@ Using `akka-discovery-kubernetes-api` is very simple, as you simply need to depe
 
 And configure it to be used as default discovery implementation in your `application.conf`:
 
+* Set `akka.management.cluster.bootstrap.contact-point-discovery.discovery-method` to `kubernetes-api`
+* For bootstrap it is recommended to set the service discovery method via `akka.management.cluster.bootstrap.contact-point-discovery.discovery-method` rather then overriding the global service discovery mechanism with `akka.discovery.method`
```

Comment:
Maybe this could be explained a little bit more:

```suggestion
* For bootstrap it is recommended to set the service discovery method via `akka.management.cluster.bootstrap.contact-point-discovery.discovery-method` rather then overriding the global service discovery mechanism with `akka.discovery.method`
* Most Kubernetes deployments should use [`akka-dns`](https://doc.akka.io/docs/akka/current/discovery/index.html#discovery-method-dns) as the default `akka.discovery.method`, which will use DNS SRV lookups for service-to-service discovery
```